index.php

```diff
@@ -6,4 +6,4 @@
  * @since 1.2.1
  */
 
-?>
+echo '';
```

Comment:
Yeah, getting this file to pass validation was...tricky. It is an empty file, but it needs to exist - so it needs structures like a FileComment. I went through several iterations looking for something that would pass muster all around, and eventually settled on simply echoing an empty string.